fix_certs.service needs to be wiped during make clean

**Describe the bug**
`fix_certs.service` isn't wiped during `make clean`/`host_cleanup.sh`.  Subsequent Metal3 deployments fail.

**To Reproduce**
Observed running d5372452cbd015ba24726b40d8ad7d54447c5623

1. Deploy a cluster successfully
2. `make clean`
3. `make`

**Observed behavior**
```
level=debug msg="module.bootstrap.libvirt_ignition.bootstrap: Destruction complete after 0s"
level=debug
level=debug msg="Destroy complete! Resources: 3 destroyed."
level=warning msg="FIXME! Exiting after bootstrap, remove when all operators will come up successfully."
++ dirname ./06_create_cluster.sh
+ sudo systemd-run --on-active=30s --on-unit-active=30m --unit=fix_certs.service ./fix_certs.sh
Failed to start transient timer unit: Unit fix_certs.service already exists.
make: *** [ocp_run] Error 1
```

--- a/ocp_cleanup.sh
+++ b/ocp_cleanup.sh
@@ -6,6 +6,7 @@ source common.sh
 source ocp_install_env.sh
 
 sudo systemctl stop fix_certs.timer
+systemctl is-failed fix_certs.service >/dev/null && sudo systemctl reset-failed fix_certs.service
 
 if [ -d ocp ]; then
     $GOPATH/src/github.com/openshift-metalkube/kni-installer/bin/kni-install --dir ocp --log-level=debug destroy bootstrap

